app/src/main/cpp/skyline/kernel/memory.cpp

```diff
@@ -62,7 +62,7 @@ namespace skyline::kernel {
             throw exception("Cannot find a suitable carveout for the guest address space");
 
         auto result{mmap(reinterpret_cast<void *>(base.address), base.size, PROT_NONE, MAP_FIXED | MAP_ANONYMOUS | MAP_PRIVATE, -1, 0)};
-        if (result == MAP_FAILED) [[unlikely]]
+        if (result == MAP_FAILED)
```

Comment:
I don't understand why you removed this?

Reply:
It's implicit for any paths that throw an exception, the compiler automatically understands that they're unlikely branches